packages/web-app-files/src/components/PublicLinksSidebar/LinkActions.vue

```diff
@@ -52,14 +52,36 @@ export default {
   },
 
   methods: {
+    ...mapActions(['showMessage', 'createModal', 'hideModal']),
     ...mapActions('Files', ['removeLink']),
     ...mapMutations('Files', ['TRIGGER_PUBLIC_LINK_EDIT']),
 
     $_removeLink() {
+      const modal = {
+        variation: 'danger',
+        title: this.$gettext('Delete this public link'),
```

Comment:
Could you add a message to the modal? Looks broken without a message to be honest. ;-) Title could be shorter (`Delete public link`, without the `this`). Message could be similar to deleting files/folders: `Are you sure you want to delete this public link?`. Maybe even add something that it's not revertible.

Reply:
Sure, good idea!